cmd/thanos/sidecar.go

```diff
@@ -100,7 +102,7 @@ func runSidecar(
 	objStoreConfig *pathOrContent,
 	peer cluster.Peer,
 	reloader *reloader.Reloader,
-	component string,
+	component component.StoreAPI,
```

Comment:
`SourceStoreAPI` technically (: It's not needed true, but techincally sidecar is both source and storeAPI so maybe worth to keep same convenience.

Reply:
That was a typo yeah. Fixed !